src/libs/ReportActionsUtils.ts

```diff
@@ -1326,7 +1330,9 @@ function getReportActionMessageText(reportAction: OnyxEntry<ReportAction> | Empt
     if (!Array.isArray(reportAction?.message)) {
         return getReportActionText(reportAction);
     }
-    return reportAction?.message?.reduce((acc, curr) => `${acc}${getTextFromHtml(curr?.html)}`, '') ?? '';
+    // Sometime html can be an empty string
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+    return reportAction?.message?.reduce((acc, curr) => `${acc}${getTextFromHtml(curr?.html ?? curr?.text)}`, '') ?? '';
```

Comment:
Here comment added but change of  ?? to || missed. Can you push these changes quickly? @nkdengineer

Reply:
Sure.